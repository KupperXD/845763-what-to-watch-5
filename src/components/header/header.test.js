import React from 'react';
import renderer from 'react-test-renderer';
import {BrowserRouter} from "react-router-dom";

import Header from "./header";

const component = () => {
  return (
    <div/>
  );
};

it(`renders correctly`, () => {
  const header = renderer
    .create(
        <BrowserRouter>
          <Header
            className={``}
            breadcrumbs={component}
            title={``}
            user={{}}
          />
        </BrowserRouter>
    )
    .toJSON();

  expect(header).toMatchSnapshot();
});
