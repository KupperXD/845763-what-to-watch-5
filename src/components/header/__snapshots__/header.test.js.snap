// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<header
  className="page-header "
>
  <div
    className="logo"
  >
    <a
      className="logo__link logo__link--undefined"
      href="/"
      onClick={[Function]}
    >
      <span
        className="logo__letter logo__letter--1"
      >
        W
      </span>
      <span
        className="logo__letter logo__letter--2"
      >
        T
      </span>
      <span
        className="logo__letter logo__letter--3"
      >
        W
      </span>
    </a>
  </div>
  <h1
    className="page-title user-page__title"
  >
    
  </h1>
  <div
    className="user-block"
  >
    <a
      className="user-block__link"
      href="/login"
      onClick={[Function]}
    >
      Sign in
    </a>
  </div>
</header>
`;
